faros_event.sh

```diff
@@ -823,30 +871,25 @@ function resolveCDInput() {
     if ! [ -z ${deploy_uri+x} ] || ! [ -z ${FAROS_DEPLOY+x} ]; then
         parseDeployUri
     fi
-    if ! [ -z ${artifact_uri+x} ] || ! [ -z ${FAROS_ARTIFACT+x} ]; then
-        parseArtifactUri
-    fi
-    if ! [ -z ${commit_uri+x} ] || ! [ -z ${FAROS_COMMIT+x} ]; then
-        parseCommitUri
-    fi
+    deploy_id=${deploy_id:-$FAROS_DEPLOY_ID}
+    deploy_app=${deploy_app:-$FAROS_DEPLOY_APP}
+    deploy_env=${deploy_env:-$FAROS_DEPLOY_ENV}
+    deploy_source=${deploy_source:-$FAROS_DEPLOY_SOURCE}
```

Comment:
so what takes precedence `--deploy <uri>` or the separate fields?

Reply:
The uri will take precedence.